zoobc/zoobc-core
# 1456 ethereum account address

## Description
Adding support for Ethereum signature.

## Breakdown
- [x] Add new EthereumAccountType and its unit test
- [x] Update account type and signature typebuf generated file

## Reference Issue
Close #1456

## Step to Test (optional)
- Run a node
- Submit a transaction signed by ethereum algorithm
- Observe that the transaction is accepted

Schema PR: https://github.com/zoobc/zoobc-schema/pull/180
